packages/web/public/static/locales/en/plumo.json

```diff
@@ -1,5 +1,5 @@
 {
-  "pageTitle": "Espero: the Plumo Ceremony",
+  "pageTitle": "The Plumo Ceremony",
```

Comment:
are we calling it the Plumo ceremony moving forward?

Reply:
I believe so. Espero, in my mind, is the setup system, which can be used for Plumo and other SNARKs as well.